src/content/self-hosted/install/releases.mdx

```diff
@@ -5,6 +5,45 @@ sidebar_label: Release Notes
 id: releases
 ---
 
+## 1.6.0
+13 March, 2023
+
+### Deprecation Notice
+- Support for Kubernetes [1.21](https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG/CHANGELOG-1.21.md) has been deprecated and will be removed next release.
+
+### Features
+
+- Add support for Kubernetes [1.24](https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG/CHANGELOG-1.24.md)
+- Update the Okteto CLI version to [2.13.0](https://github.com/okteto/okteto/releases/tag/2.13.0)
+- Default ingress class for ingresses managed by okteto is `okteto-nginx`. All ingresses managed by okteto are migrated automatically during the upgrade
+- Support [custom quotas](self-hosted/administration/configuration.mdx#quotas) for node port and load balancer service types
+- Public override support for self-signed certificates and bring your own certificate
+- Wake operation takes into account dependencies between dev environments to wake dependant environments first
+
+### Improvements
+
+- Several improvements in the UI to keep the status updated accordingly with the status in user's dev environments
+- Allow users to access to namespaces that are being terminated to see the operation logs
+- Added input autofocus in several dialogs
+- Improve experience in "New namespace" dialog to enable faster the button
+- Filter branches results by search term in GitHub deploy dialog
+- Remove redundant headers in "Settings" view
+- Do not autodestroy GC job when it ends
+- Private endpoints cookie is now set in the backend and specified as "httpOnly"
+- GC deletes pods stuck in `Failed` phase with `NodeAffinity` reason
+- Upgrade Reloader dependency to [1.0.5](https://github.com/stakater/Reloader/releases/tag/v1.0.5)
+
+### Bugfixes
+
+- Skip volume snapshots deletion as part of destroy all operation if "include volumes" check is not selected
+- Fix "Destroy all", "Delete namespace" and "Wake namespace" operations when using private CAs or self-signed certificates
+- Display the correct operation name in logs when deleting a namespace or destroying all resources within a namespace
+- Avoid duplicated log lines when the browser visibility changes
+- Self-signed certificate is recreated when subdomain or public override changes after an upgrade
```

Comment:
If we are talking about the property, use publicOverride. And can we link to the corresponding config logs?

Reply:
Sure, changed